src/main/scala/io/iohk/ethereum/network/rlpx/RLPxConnectionHandler.scala

```diff
@@ -151,14 +152,20 @@ class RLPxConnectionHandler(nodeKey: AsymmetricCipherKeyPair)
       }
 
     def handleWriteFailed: Receive = {
-      case CommandFailed(_: Write) =>
-        log.warning("Write failed")
+      case CommandFailed(cmd: Write) =>
+        log.warning(s"[Stopping Connection] Write to peer $peerId failed, trying to send ${Hex.toHexString(cmd.data.toArray[Byte])}")
```

Comment:
yes it can, but I think we want to have as much information about failure as possible, also if it is big you will immediately find it scrolling through log, we are zipping log files after all so file size is not the issue